src/theme/book.js

```diff
@@ -161,6 +161,10 @@ $( document ).ready(function() {
             ace_theme = "ace/theme/dawn";
         }
 
+        setTimeout(function() {
```

Comment:
The timeout is needed to allow the browser to apply the stylesheet before extracting the background color.

Reply:
Would this mean it's possible for the menu bar to (very quickly) flash white, kinda like the flash-of-unstyled-text issue you sometimes get with CSS?

Reply:
Possible, but Chrome applies a subtle transition effect and I have not found it to be an issue.

Reply:
So it might be white for a very brief period when the page is first loaded. After that in changes instantly.